enterprise/reporting/event_sampler/stats_collector.go

```diff
@@ -9,9 +9,11 @@ import (
 const (
 	StatReportingEventSamplerRequestsTotal   = "reporting_event_sampler_requests_total"
 	StatReportingEventSamplerRequestDuration = "reporting_event_sampler_request_duration_seconds"
+	StatReportingBadgerDBSize                = "reporting_badger_db_size"
```

Comment:
Is the size in bytes ?

```suggestion
	StatReportingBadgerDBSize                = "reporting_badger_db_size_bytes"
```

Reply:
yes, it is in bytes